todo_app/data/api.py

```diff
@@ -0,0 +1,114 @@
+from configparser import ConfigParser
+from todo_app.flask_config import Config
+from .boards import *
+import os
+import requests
+
+
+class TrelloAPI():
+    
+    def __init__(self):
+        self.TRELLO_URL = 'https://api.trello.com/1'
+        self.key_and_token = {'key':Config().TRELLO_KEY,'token':Config().TRELLO_TOKEN}
+        self.boardId = Config().BOARD_ID
+        self.to_do_id = Config().TO_DO_ID
+        self.doing_id = Config().DOING_ID
+        self.done_id = Config().DONE_ID
+        print("API board id:")
+        print(Config().BOARD_ID)
+        print("API to do id:")
+        print(Config().TO_DO_ID)
+
+    def get_boards(self):
+        boards_url = self.TRELLO_URL + '/members/me/boards'
+        key_and_token = self.key_and_token
+        headers = {"Accept": "application/json"}
+        arguments = {'fields': 'name', 'lists': 'open'}
+        response = requests.get(boards_url, params=key_and_token, data=arguments)
+        json_response = response.json()
+        return json_response
+    
+    def create_board(self, name):
+        name = name 
+        url = "https://api.trello.com/1/boards/"
+        key_and_token = self.key_and_token
+        arguments = { 'name': name}
+        response = requests.request( "POST", url, params=key_and_token, data=arguments)
+        json_response = response.json()
+
+        print(json_response)
+        return json_response
+    
+    def delete_board(self, board_id):
+        board_id = board_id 
+        url = "https://api.trello.com/1/boards/" + board_id
+        key_and_token = self.key_and_token
+        response = requests.delete(url, params=key_and_token)
+       
+    def create_list(self, name, board_id):
+        url = "https://api.trello.com/1/lists"
+        key_and_token = self.key_and_token
+        arguments = { 'name': name , 'idBoard': board_id}
+        response = requests.request( "POST", url, params=key_and_token, data=arguments)
+        json_response = response.json()
+
+        print(json_response)
```

Comment:
Similarly with these responses that we print out

Reply:
Done